src/applications/personalization/profile360/components/Vet360EditModal.jsx

```diff
@@ -89,7 +90,7 @@ export default class Vet360EditModal extends React.Component {
             onDelete={onDelete}
             title={title}
             analyticsSectionName={analyticsSectionName}
-            deleteEnabled={!isEmpty()}>
+            deleteEnabled={!isEmpty() && !deleteDisabled}>
             <LoadingButton isLoading={isLoading}>Update</LoadingButton>
```

Comment:
I feel like this could be more flexible by having `deleteDisabled` as a function instead, something like `isDeleteDisabled` that defaults to just `!isEmpty()`.

Reply:
yep we can refactor if it's being used in 2 or more places